modules/graph/src/model/advisory.rs

```diff
@@ -73,7 +73,7 @@ impl AdvisoryDetails {
 pub struct AdvisoryVulnerability {
     pub vulnerability_id: String,
     #[schema(default, value_type = Vec<String>)]
-    pub cvss3_scores: Vec<Cvss3Base>,
+    pub cvss3_scores: Vec<String>,
```

Comment:
@bobmcwhirter doesn't this obviate the serialization impl's? can i remove them? and fix utopia indentation?

Reply:
Yes!